.github/workflows/checkin.yml

```diff
@@ -0,0 +1,31 @@
+name: pull-request
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+      - uses: actions/checkout@v2
+      - name: Set up node 
+        uses: actions/setup-node@v1
+      - name: Install dependencies
+        run: |
+          id
+          whoami
+          ps -ef
+          pwd
+          sudo apt-get update -qq
+          sudo apt-get -y install -qq automake autopoint xsltproc libselinux1-dev gettext expect
+          sudo apt-get -y install -qq byacc libtool
```

Comment:
why not merge into the previous line ?  it'll prob be faster to batch them all together at once.